docs/source/community/contribution_guide.rst

```diff
@@ -30,7 +31,8 @@ here is the basic process.
       We also maintain some labels for issues that are likely to be
       good for new people, e.g., **bootcamp** and **1hr**, although
       these labels are less well maintained.
-   -  Join us on Slack and let us know you're interested in getting to
+   -  Join us on `dev discuss <https://dev-discuss.pytorch.org/>`_ 
```

Comment:
Adding the dev forum is a great addition here. But I think slack is still a better place for people to say they're looking for a task to pick up.
The forum is more of a place to ask question or read about design discussions between core devs.

Reply:
Unfortunately slack is restricted right now (people need to be manually added) and the idea was to promote the option that was available to everyone. Thoughts?

Reply:
We can iterate on this later too

Reply:
Slack is not very restricted. Every application gets accepted.

Reply:
Interesting - I was going by https://discuss.pytorch.org/t/request-to-join-slack-channel/16614 but maybe we should change that policy over time?